workflows/eager.nf

```diff
@@ -205,16 +207,30 @@ workflow EAGER {
 
     if ( !params.skip_deduplication ) {
         DEDUPLICATE( ch_reads_for_deduplication, ch_fasta_for_deduplication.fasta, ch_fasta_for_deduplication.fasta_fai )
-        ch_versions          = ch_versions.mix( DEDUPLICATE.out.versions )
-
-        ch_dedupped_bams     = DEDUPLICATE.out.bam.join( DEDUPLICATE.out.bai )
+        ch_dedupped_bams = DEDUPLICATE.out.bam
+            .join( DEDUPLICATE.out.bai )
         ch_dedupped_flagstat = DEDUPLICATE.out.flagstat
+        ch_versions                   = ch_versions.mix( DEDUPLICATE.out.versions )
 
     } else {
         ch_dedupped_bams     = ch_reads_for_deduplication
         ch_dedupped_flagstat = Channel.empty()
     }
 
+    //
+    // MODULE: PreSeq
+    //
+    if ( !params.mapstats_skip_preseq && params.mapstats_preseq_mode == 'c_curve') {
```

Comment:
Why two separate `if` statements? Can be combined into one `if/else`

Reply:
Agree

Reply:
Agree